src/header/profile.tsx

```diff
@@ -173,33 +182,33 @@ export default class Profile extends PureComponent<ProfileProps> {
     const items = [
       showApplyChangedUser && {
         rgItemType,
-        label: translations.applyChangedUser || 'Apply changed user',
+        label: translations?.applyChangedUser ?? messages.applyChangedUser,
```

Comment:
What do you think about using the same style everywhere?
```suggestion
        label: (translations ?? messages).applyChangedUser,
```

Reply:
Good point, thanks!

Reply:
Addressed in https://github.com/JetBrains/ring-ui/pull/5542